src/renderers/index.ts

```diff
@@ -31,7 +32,8 @@ export
 class HTMLWidget extends Widget {
   constructor(html: string) {
     super();
-    this.node.innerHTML = html;
+    let range = document.createRange();
+    this.node.appendChild(range.createContextualFragment(html));
```

Comment:
Oh, sorry, confused two different issues. We found a problem with Safari (latest version) where calling `createContextualFragment` would throw a DOM exception, in certain situations (whereas latest Chrome and Firefox worked fine with same code). So there seem to still be some implementation differences with this experimental API.